WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -69,17 +69,35 @@ private static EstimateSmartFeeResponse SimulateRegTestFeeEstimation(int confirm
 		/// </summary>
 		public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc, EstimateSmartFeeMode estimateMode = EstimateSmartFeeMode.Conservative, bool simulateIfRegTest = false)
 		{
-			var estimations = (simulateIfRegTest && rpc.Network == Network.RegTest)
+			var smartEstimations = (simulateIfRegTest && rpc.Network == Network.RegTest)
 				? SimulateRegTestFeeEstimation(estimateMode)
 				: await GetFeeEstimationsAsync(rpc, estimateMode).ConfigureAwait(false);
 
-			var rpcStatus = await rpc.GetRpcStatusAsync(CancellationToken.None).ConfigureAwait(false);
 			var mempoolInfo = await rpc.GetMempoolInfoAsync().ConfigureAwait(false);
 			var sanityFeeRate = mempoolInfo.GetSanityFeeRate();
+			var rpcStatus = await rpc.GetRpcStatusAsync(CancellationToken.None).ConfigureAwait(false);
+
+			if (mempoolInfo.Histogram is { })
```

Comment:
Just mentioning that even with Bitcoin core this is not null, instead of an {NBitcoin.FeeRateGroup[0]}. So having the condition here makes no sense.